src/test/groovy/com/github/vkorobkov/jfixtures/integration/H2IntegrationTest.groovy

```diff
@@ -0,0 +1,32 @@
+package com.github.vkorobkov.jfixtures.integration
+
+import com.github.vkorobkov.jfixtures.testutil.H2Test
+import spock.lang.Specification
+
+import java.nio.file.Path
+
+class H2IntegrationTest extends Specification implements H2Test {
+    Path tmpFolderPath = unpackYamlToTempFolder("default.yml")
+
+    void setupSpec() {
+        sql.execute("DROP TABLE IF EXISTS users")
```

Comment:
Maybe we'll need some pretty API in `H2Test` trait for dropping and creating a new table.

Would be nice just to call a methods like: `recreateTable("users", [ID: "INT PRIMARY KEY", NAME: "varchar(50) DEFAULT NULL", AGE: "GE INT DEFAULT NULL")`